runtime-library/CMakeLists.txt

```diff
@@ -15,13 +15,16 @@ set(IREE_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../iree" CACHE STRING "Main IR
 option(IREERT_ENABLE_LTO "Enable LTO (link time optimization) if supported" ON)
 include(CheckIPOSupported)
 check_ipo_supported(RESULT _ireert_lto_supported OUTPUT error)
-if(IREERT_ENABLE_LTO)
-  if(_ireert_lto_supported)
-    message(STATUS "Enabling LTO")
-    set(CMAKE_INTERPROCEDURAL_OPTIMIZATION ON)
-  else()
-    message(WARNING "LTO not supported by toolchain bit requested (ignored)")
-  endif()
+if(NOT _ireert_lto_supported)
```

Comment:
wdyt about doing this instead:

```
if(APPLE)
  # ... comments about LTO support
  set(_ireert_lto_supported FALSE)
endif()
```

And then just take the original path through above?